assets/styl/_modules/_media.styl

```diff
@@ -20,7 +20,7 @@
                 fill rgba(255, 255, 255, .5)
         ') // @stylint ignore
         background-size 65% 65%
-        background-position center center
+        background-position top 40% left 50%
```

Comment:
This value is not standard https://developer.mozilla.org/fr/docs/Web/CSS/background-position